lib/normalizer.c

```diff
@@ -3488,6 +3488,19 @@ grn_nfkc_normalize_unify_iteration_mark(grn_ctx *ctx,
     return unified_buffer;
   }
 #  undef N_KATAKANA_BYTES
+#  define N_KANJI_BYTES 3
+  else if (current_length == 3 && current[0] == 0xe3 && current[1] == 0x80 &&
+           current[2] == 0xbb && previous_length == N_KANJI_BYTES &&
```

Comment:
Can we assume that all Kanji characters use 3 bytes?

For example: 𠀋 U+2000B CJK UNIFIED IDEOGRAPH-2000B